tests/test_geoprocessing.py

```diff
@@ -2624,7 +2625,8 @@ def test_convolve_2d_multiprocess(self):
         # the sides and realizing diagonals got subtracted twice
         expected_result = test_value * (
             n_pixels ** 2 * 9 - n_pixels * 4 * 3 + 4)
-        self.assertAlmostEqual(numpy.sum(target_array), expected_result)
+        self.assertAlmostEqual(numpy.sum(target_array), expected_result,
+                              places=3)

```

Comment:
Is this indent off by one space too?

```suggestion
                               places=3)
```

Reply:
It sure is!  Not sure how that happened.  Patched in d916244.